netlify.toml

```diff
@@ -4,3 +4,8 @@
   base = "code/"
   publish = "build/"
   command = "npm run build"
+
+[[redirects]]
+  from = "/*"
```

Comment:
Pretty cool that you settled this issue yourself even before Max's support ! Well done!